src/Core/Theme.re

```diff
@@ -51,10 +62,12 @@ type editorGutter = {
 };
 
 type t = {
-  background: Color.t,
```

Comment:
Remove this because it's not an actual theme color. We might want to use `panel.background` instead, but I've just used `editor.background` so far, which seems to be fine.

Reply:
Thanks @glennsl !